.eslintignore

```diff
@@ -3,9 +3,7 @@
 /docs/export
 /lerna.json
 /packages/demo-app/public/static-data
-/packages/grid/lib/lodash
-/packages/grid/x-grid-modules/lib/autosizer
-/packages/grid/x-grid-modules/lib/lodash
+lib
```

Comment:
This is the ignore file for eslint.
So we just tell eslint to ignore any lib folder as we don't own the code in this folder, it just contains 3rd party libraries.
Yes we can refactor but we will move the code to source and delete it from this folder if we do

Reply:
it's also slighlty different as it has types mocha and root tsconf has type jest.
IMHO we should move everything to jest ts and get rid of karma mocha chai...

Reply:
> IMHO we should move everything to jest ts and get rid of karma mocha chai...

I think that this can happen once we successfully can get an alternative that delivers significantly (to make the opportunity cost worth it) **more value** than whatever already exists. Some of the prerequisites I think that we should expect from tests:

1. that can run in the CI
2. that is reliable (no flaky)
3. that is fast
4. that allows us to run in different browsers, e.g. #316
5. that has a great test coverage (you can test most of what you need to):

So far, we have identified Jest + Puppeteer as a path that: can deliver on 5., 1. should be possible 2. will be a bit challenging 3. might be challenging 4. is impossible by design.